packages/sshnoports/lib/sshnp/sshnp.dart

```diff
@@ -63,6 +63,11 @@ abstract class SSHNP {
   /// is set to '0' then
   abstract String localPort;
 
+  /// Port that local sshd is listening on localhost interface
+  /// Default set to 22
+
+  abstract String localSshdPort;
```

Comment:
Would prefer this to be an int, since int.parse could be a non-numeric String, and that is extra validation that needs to be done by the sshnp class

Reply:
I set as a String as this is what the Params give but thought the same as I was writing the code... Prefer int too.. Where is the best place the do that conversion ?

Reply:
In the meantime added checks for int's with int.tryParse and port range checks